### DISCUSSION TITLE
Update on cell change

### DISCUSSION THREAD

**Discussion by WiFiSaltacoda** [sqlitebrowser/sqlitebrowser#3314] [Q&A]
Hello,
Is there an option that allows you to save a change, made directly in a table from "Browse Data", on cell change, without having to press "Write Changes" (like MSAcces does)?
Any and all help would be greatly appreciated!
Thx

**Comment by chrisjlocke** [comment]
Unfortunately not.
SQLite databases allow connections from a whole host of programming languages, so you could use another programming language to make the change.
You could even use our API if the database was hosted on DBHub.io and use a simple curl script to update the database.
